imports/startup/server/accounts.js

```diff
@@ -32,6 +32,24 @@
  * admin roles on a fresh install (when number of users is zero)
  */
 
+//Generating 8 Length Alphanumeric Password.
+function passwordGen(len){
+    var text = " ";
+    var charset = "abcdefghijklmnopqrstuvwxyz0123456789";
+    for( var i=0; i < len; i++ )
+  text += charset.charAt(Math.floor(Math.random() * charset.length));
+    return text;
+}
+
+//Applying Password to Meteor Admin User Settings and Logging to Server.
+var randomPassword = passwordGen(8)
+var newPassword = randomPassword.substring(1)
+
+//User ID for Admin is currently bMfEoAFqu4EibGwtH
+console.log("Admin Password: " + newPassword);
+Accounts.setPassword("bMfEoAFqu4EibGwtH",newPassword)
```

Comment:
We can't use database id in code, it's automatically generated, and is different every time.

Remove code at line number 46, 49, 50. Setting password will work fine with logic given in my comment below.